conn.go

```diff
@@ -136,6 +136,15 @@ func NewConnWith(conn net.Conn, config ConnConfig) *Conn {
 	return c
 }
 
+// DeleteTopics deletes the specified topics.
+func (c *Conn) DeleteTopics(topics ...string) error {
+	_, err := c.deleteTopics(deleteTopicsRequestV1{
+		Topics:  topics,
+		Timeout: int32(30 * time.Second / time.Millisecond),
```

Comment:
I agree that this 30 sec timeout is not perfect. It's consistent with `CreateTopics` (please see [here](https://github.com/segmentio/kafka-go/blob/master/createtopics.go#L274)), but I'd be happy to fix both `DeleteTopics` and `CreateTopics`. 

I think using a negative timeout can cause issues like this:

```go
conn.CreateTopics("newtopic")

// This will probably fail as the topic has not been created yet:
doSomethingWithTopic("newtopic") 
```

So I'd prefer to use the connection's write deadline if that's ok.

Reply:
Fixed

Reply:
Sounds good.

I missed that we introduced that 30s hard coded timeout on CreateTopics, probably something that we'll have to fix eventually.

Using the connection's deadline sound fine to me 👍

Reply:
I've just fixed that for both `DeleteTopics` and `CreateTopics`